<launch>
    <!-- Offical SDK -->
    <arg name="config_file" value=".../robotcar-dataset-sdk"/>
    <!-- Dataset Path -->
    <arg name="dataset_path" value=".../2019-01-15-13-06-37-radar-oxford-10k"/>
    <!-- Save flag: 0 No, 1 Yes -->
    <arg name="save_flag" value="0"/>
    <!-- Save Path -->
    <arg name="save_path" value=".../2019-01-15-13-06-37-radar-oxford-10k.bag" />
  
    <!-- Nodes -->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find radar_robotcar_dataset_ros)/launch/rviz/radar_robotcar_dataset_ros.rviz"/>
    <node name="oxford_sensors" pkg="radar_robotcar_dataset_ros" type="oxford_sensors" args="$(arg config_file) $(arg dataset_path) $(arg save_flag) $(arg save_path)" respawn ="false"/>
    <node pkg="mapviz" type="mapviz" name="mapviz"></node>
    <node pkg="swri_transform_util" type="initialize_origin.py" name="initialize_origin" >
      <param name="local_xy_frame" value="/map"/>
      <param name="local_xy_origin" value="auto"/>
      <rosparam param="local_xy_origins">
        [{ name: swri,
           latitude: 29.45196669,
           longitude: -98.61370577,
           altitude: 233.719,
           heading: 0.0},
         
         { name: back_40,
           latitude: 29.447507,
           longitude: -98.629367,
           altitude: 200.0,
           heading: 0.0}]
      </rosparam>
      <remap from="fix" to="/oxford/gps/gps"/>
    </node>
    <node pkg="tf" type="static_transform_publisher" name="oxford_transform" args="0 0 0 0 0 0 /map /Bumblebee 100"  />


</launch>